### DISCUSSION TITLE
Config file for Docker

### DISCUSSION THREAD

**Discussion by darrepac** [abraunegg/onedrive#1482] [Q&A] [answered]
Hi

Could elaborate on how to pass the config file into docker? do I have to create it inside docker image bash? where?

thanks

**Comment by abraunegg** [comment, marked as answer]
@darrepac 
This is articulated in the Docker documentation.

The configuration for the client resides in the configuration volume:
```
onedrive_conf:/onedrive/conf
```

Refer to https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md#6-edit-the-config to modify this configuration.

**Reply by darrepac** [reply]
> Refer to https://github.com/abraunegg/onedrive/blob/master/docs/Docker.md#6-edit-the-config to modify this configuration.

Was missing that, thanks!
